docs/source/changelog.rst

```diff
@@ -17,6 +17,20 @@ All notable changes to this project will be documented in this file.
 
 We have listed major changes for each release below.
 
+3.3.1
+==============================================================================
+
+- **Release date:** 22nd April 2020
+- **Downloads:** `Source downloads are available on GitHub <https://github.com/uccser/cs-field-guide/releases/tag/3.3.1>`__
+
+**Changelog:**
+
+- Set crossorigin attribute on the default image in pixel viewer. `#1254 <https://github.com/uccser/cs-field-guide/pull/1254>`__
```

Comment:
How about something more user-readable, like, solved bug in pixel viewer that was affecting some people